### ISSUE TITLE
Fermata auto 1.9.8  Andorid 14 ,  One UI 6.0 Error

### ISSUE THREAD

**Issue by jaemyeong2** [AndreyPavlenko/Fermata#373]
After updating to Android 14, One UI 6.0, and using AAWireless, I frequently encounter the issue of no sound in YouTube. Is there a solution for this?

**Comment by AndreyPavlenko** [comment]
There are some audio settings in AAWireless. Have you tried?

**Comment by jaemyeong2** [comment]
Of course! I proceeded with the settings within the app's functionalities.

**Comment by rgfelipe123** [comment]
Tudo funcionando
Precisa downgrade para ui5 via odin
Instalar fermata auto via aaad
Upgrade ui6 via odin mantendo home csc como padrão
Funciona tudo
Firmware obter em sammobile

**Comment by mik1011984** [comment]
Thank you for adding screen mirroring. So far it works great. The screen on my phone is on the small side so I don't see much difference between the two mirroring options other than the control scheme. One  method feels quite a bit slower than the other, but I'll play around with it some more. Thanks again for your hard work.
We're one step closer to eliminating the need for Screen2Auto, which hasn't been updated in a long time and can be more finicky for users to get working and stay working (depending on their setup).

**Comment by Moumyio** [comment]
> Tudo funcionando Precisa downgrade para ui5 via odin Instalar fermata auto via aaad Upgrade ui6 via odin mantendo home csc como padrão Funciona tudo Firmware obter em sammobile

Você conseguiu fazer o downgrade ui6->ui5 via odin mantendo home csc?

**Comment by rgfelipe123** [comment]
Sim
Tem que fazer com firmware da sammobile
Manter googleplayservices sem atualização automática
Minha versão do androide auto é 11

Em dom., 4 de fev. de 2024, 14:21, Moumyio ***@***.***>
escreveu:

**Comment by rgfelipe123** [comment]
Fazer backup antes
Instalar ui5 via odin
Instalar fermata auto via aaad
Atualizar para 1.9.8 no github
Instalar androide auto 9.xx
Ver se tá funcionando
Bloquear atualizações automáticasprincipalmente Google play services
Instalar ui6 com homens
Tem que ser firmware baixado do Sam mobile
Pode ver o vídeo
No carro funciona até mirror


Em seg., 5 de fev. de 2024, 11:28, rogerio felipe <
***@***.***> escreveu:

**Comment by rgfelipe123** [comment]
Downgrade
Usar csc no firmware do ui5
Restaurar seus dados e aplicativos
Depois upgrade do ui6 com homens
Bloquear atualizacao automática do googleplay

**Comment by rgfelipe123** [comment]
Uploading VID-20240206-WA0004.mp4…

**Comment by rgfelipe123** [comment]
https://github.com/AndreyPavlenko/Fermata/assets/101208200/df1a20a5-f547-4f0d-859f-b31ac5ce2ad8

**Comment by Moumyio** [comment]
> Downgrade Usar csc no firmware do ui5 Restaurar seus dados e aplicativos Depois upgrade do ui6 com homens Bloquear atualizacao automática do googleplay


Resultou! Obrigado, rgfelipe123!
--

I used following steps on my Galaxy S21:
1. Backup user data
2. Downgrade OneUI 6.0 -> 5.1 via Odin with factory reset (CSC instead of HOME CSC).
3. Install AA+10.9.634643 and KingInstaller.v1.4
4. Install Fermata Auto via KingInstaller.v1.4, disable auto update in fermata settings
5. Wireless update OneUI 5.1 to the latest version OneUI 6.0
6. Restore user data
7. Allow all updates updates via Google Play incl. AA
8. From now on, NO attempts to update Fermata Auto!

**Comment by soapdishbandit** [comment]
Any chance to wait and just see if new version will support Android 14? Mine worked fine even with Android 14 and One UI 6.0, but then I updated....

**Comment by project-blaze** [comment]
Hi, my phone is S24, and Fermata Auto doesn't appear on the Android Auto launcher. I also encounter the same issue when I go to the YouTube page; the screen keeps flickering or resetting to the top, even when I try to scroll down. However, on the Android Auto launcher, only the Fermata Auto controller is loaded.

**Comment by rgfelipe123** [comment]
Downgrade to ui5.1 firmware in sammobile
Install fermata auto 1.9.8
Upgrade to ui6.0 usina firmware sammobile
Block New updates jn Google play
Only this
My Phone is S21  ui6.0 and fermata everthing  works

Em qui., 8 de fev. de 2024, 23:09, soapdishbandit ***@***.***>
escreveu:

**Comment by vocaloo** [comment]
> Tudo funcionando Precisa downgrade para ui5 via odin Instalar fermata auto via aaad Upgrade ui6 via odin mantendo home csc como padrão Funciona tudo Firmware obter em sammobile


This manual is for use by cable or wireless. I have the same cell phone

**Comment by rgfelipe123** [comment]
Cabe by pc

Em sáb., 10 de fev. de 2024, 22:53, vocaloo ***@***.***>
escreveu:

**Comment by mpeoeoeo** [comment]
Is there currently any way to use Fermata Auto on Android OS 14 One UI 6.0 version without downgrade the version?

**Comment by rgfelipe123** [comment]
I dont know
But the problem is automatic update in googleplay

Em qua., 6 de mar. de 2024, 04:26, mpeoeoeo ***@***.***>
escreveu:
